### PR TITLE
Update variable name

### PR DESCRIPTION
**Pull request by hannahramadan**
`github_diff` should be`github_diff_exist` so that the variable is used vs the method. This has been updated.

### CODE REVIEW

**Review by kaylareopelle — COMMENTED** [review]
Nice! Is there a test change that should accompany this fix to prevent future regressions?

**Comment by hannahramadan** [comment]
> Nice! Is there a test change that should accompany this fix to prevent future regressions?

Looks like that method isn't being tested at all. Will work on that!

File: test/new_relic/slack_notifications_tests/gem_notifier_tests.rb [hunk 1]
```diff
@@ -113,4 +118,24 @@ def test_interpolate_rubygems_url
...
+
+  def test_gem_message_incorrect_args
+    GemNotifier.stub(:abort, nil) do
+      assert_raises(StandardError) { GemNotifier.gem_message('fake_gem', '1.2') }
+    end
+  end
+
+  def test_gem_message_with_github_diff
+    GemNotifier.stub(:github_diff, 'fake_gem_diff_url') do
+      assert_equal "A new gem version is out :sparkles: <https://rubygems.org/gems/fake_gem|*fake_gem*>, 1.1 -> 1.2\n\n<fake_gem_diff_url|See what's new.>",
```
[the remark this answers is not in the document]
Comment (kaylareopelle) [reply test/new_relic/slack_notifications_tests/gem_notifier_tests.rb:130]: You could try breaking it up using `\` characters: https://stackoverflow.com/a/10524123/16434014

[the remark this answers is not in the document]
Comment (hannahramadan) [reply test/new_relic/slack_notifications_tests/gem_notifier_tests.rb]: Worked perfectly thank you: https://github.com/newrelic/newrelic-ruby-agent/pull/1507/commits/cb8620804f735068539f1417e19ed344bd094faa

**Review by kaylareopelle — APPROVED** [review]
🎉

**Pull request merged**